docs/_snippets/main-config-indexers-title.md

```diff
@@ -6,6 +6,7 @@ const combosIndexer = {
     const title = fileName.match(/\/(.*)\.stories/)[1];
 
     // Read file and generate entries ...
+    const entries = [];
 
     return entries.map((entry) => ({
```

Comment:
Does this need the `entries.map` at all? As it is, it's always mapping over an empty array and therefore doing nothing. Maybe it should be:

```js
return {
  type: 'story',
  // 👇 Use makeTitle to format the title
  title: `${makeTitle(title)} Custom`,
  importPath: fileName,
  exportName: entry.name,
};
```

Reply:
```suggestion
    let entries = [];
    // Read file and generate entries...

    return entries.map((entry) => ({
```